### PR TITLE
replace azure with new packages

### PR DESCRIPTION
**Pull request by freezeboy**
Related to #1078

### CODE REVIEW

**Comment by Odianosen25** [comment]
Hello @freezeboy,

Thanks for this, and its something I will look into now once I can get my head around it.

@basnijholt please you did help in integrating the original azure checks. Can you kindly help in confirming this, and assist in letting us know if there are any other works to be done outside this.

Regards

**Review by basnijholt — APPROVED** [review]
LGTM!

File: requirements.txt [hunk 1]
```diff
@@ -16,6 +16,10 @@ python-socketio==4.4.0
 deepdiff==4.3.1
 python-dateutil==2.8.1
 pid==2.2.5
-azure==4.0.0
+azure-mgmt-compute
```
Comment (basnijholt) [inline requirements.txt:19]: Should probably pin the versions though, like th rest.

Comment (Odianosen25) [reply requirements.txt:19]: Thanks for confirming @basnijholt, much appreciated.

@freezeboy, as suggested by Bas,  can you kindly have them pinned so I can merge once all checks passed?

Regards

**Comment by Odianosen25** [comment]
@freezeboy,

Please can you pin the packages so this can be merged?

It will help resolve some build issues we experiencing. 
Regards

**Pull request merged**